### DISCUSSION TITLE
Broken background during open dialog

### DISCUSSION THREAD

**Discussion by Kowersky** [zauberzeug/nicegui#3572] [Q&A]
### Question

What can be done to keep this gui from breaking in the background?

https://github.com/user-attachments/assets/72dbffae-c626-4e82-bb26-14e2288ee833

**Comment by falkoschindler** [comment]
Sorry, I don't see what you mean with "breaking in the background". What is happening? What did you expect instead?
And can you, please, try to create a [minimum reproducible example](https://github.com/zauberzeug/nicegui/wiki/FAQs#why-is-it-important-to-provide-a-minimal-reproducible-example)? Sometimes code speaks more than words.

**Reply by Kowersky** [reply]
The background image shifts and it is supposed to be in place as it was before the dialogue was opened.
